modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/introspection/validation/ParameterModelValidatorTestCase.java

```diff
@@ -91,4 +140,63 @@ public void invalidModelDueToNoReturnType()
         when(operationModel.getParameterModels()).thenReturn(asList(invalidParameterModel));
         validator.validate(extensionModel);
     }
+
+
+    private static class InvalidPojo {
+
+        public String getName()
```

Comment:
do you actually use this setter o getter? if not, then just annotate the name with `@Parameter`